site/user-guide.md

```diff
@@ -98,12 +98,14 @@ Oracle recommends careful consideration before deciding to expose any administra
 * Flannel networking v0.9.1-amd64 (check with `docker images | grep flannel`).
 * Docker 18.03.1.ce (check with `docker version`).
 * Helm 2.8.2+ (check with `helm version`).
-* Oracle WebLogic Server 12.2.1.3.0 with patch 29135930.
+* Oracle WebLogic Server 12.2.1.3.0 with patch 29135930 (check with `docker run store/oracle/weblogic:12.2.1.3 /u01/oracle/OPatch/opatch lsinventory`).
```

Comment:
Monica:

As far as I can tell, the opatch lsinventory command won't actually report the WL version in a clear way even if adding `-details` or `-all`.  Go figure.  It doesn't seem to be designed to tell you what products are installed so much as which patches are installed.  Is there a way to invoke it to get the WL version out of it in a clear way?

Alternatively, the following will report the version and the patches:    `docker run store/oracle/weblogic:12.2.1.3 sh -c 'source /u01/oracle/wlserver/server/bin/setWLSEnv.sh && java weblogic.version -verbose'`, so maybe that's better?

Reply:
It is lspatches that tell you WLS version and the patches that have been applied.  
  $ cd OPatch
    $ ./opatch version
    $ ./opatch lspatches

Reply:
./opatch version is specific to the version of opatch which sometimes needs to be updated.  For this documentation lspatches should be enough.
Monica

Reply:
OK, let's change "(check with docker run store/oracle/weblogic:12.2.1.3 /u01/oracle/OPatch/opatch lsinventory)" to (check wl version with docker run store/oracle/weblogic:12.2.1.3 sh -c 'source $ORACLE_HOME/wlserver/server/bin/setWLSEnv.sh > /dev/null 2>&1 && java weblogic.version', check wl patches with docker run store/oracle/weblogic:12.2.1.3 sh -c '$ORACLE_HOME/OPatch/opatch lspatches')

Reply:
(The reason I think we need 'weblogic.version' too is that none of the opatch options - lspatches/version/lsinventory - clearly reveal the WL version.)